clients/subscription-client/subscription-api-spec.yaml

```diff
@@ -120,6 +120,9 @@ components:
           type: integer
         oracleAccountNumber:
           type: integer
+        awsAccountId:
+          type: integer
+          format: int64
```

Comment:
We should confirm exact field name, but I expect this will end up in aws externalReference.

Reply:
how can I confirm?

Reply:
Confirmed format with Jared and updated.